src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+
+C'est une autre mesure qui, je pense, a été faussée par l'utilisation des Google Fonts, car Google Fonts utilise la "plage unicode" (`unicode-range`) dans la plupart (sinon la totalité) de ses CSS. Je m'attendrais à ce que cela soit moins fréquent du côté des utilisateurs, mais peut-être qu'il sera possible de filtrer les demandes de Google Fonts dans la prochaine édition de l'Almanach.
+
+## Ne demandez pas de police web si la police système existe
+
+{{ figure_markup(
+  caption="Pourcentage de pages mobile qui déclarent une police web avec la propriété <code>local()</code>.",
+  content="59%",
+  classes="big-number"
+)
+}}
+
+`local()` est une belle façon de référencer une police système dans votre `@font-face` `src`. Si la police `local()` existe, il n'est pas du tout nécessaire de faire une requête pour une police web. Cette propriété est utilisée de manière extensive et controversée par Google Fonts, et il s'agit donc probablement d'un autre exemple de données biaisées si nous essayons de glaner des modèles à partir du terrain des utilisateurs. - du côté des utilisateurs
+
+Il convient également de noter ici que des personnes plus intelligentes que moi (Bram Stein de TypeKit) ont déclaré que <a hreflang="en" href="https://bramstein.com/writing/web-font-anti-patterns-local-fonts.html">l'utilisation de `local()` peut être imprévisible car les versions de polices installées peuvent être obsolètes et peu fiables</a>.
+
+## Polices condensées et `font-stretch`
+
+{{ figure_markup(
+  caption="Pourcentage des pages pour ordinateurs de bureau et pour mobile qui comportent un style avec la propriété <code>font-stretch</code>.",
+  content="7%",
+  classes="big-number"
+)
+}}
+
+
+Historiquement, `font-stretch` a souffert d'un mauvais support des navigateurs et n'était pas une propriété `@font-face` bien connue. En savoir plus sur [`font-stretch` sur MDN](https://developer.mozilla.org/fr/docs/Web/CSS/font-stretch). Mais <a hreflang="en" href="https://caniuse.com/#feat=css-font-stretch">le support des navigateurs</a> s'est élargi.
+
+Il a été suggéré que l'utilisation de polices condensées sur des fenêtres d'affichage plus petites permet d'afficher plus de texte, mais cette approche n'est pas couramment utilisée. Cela dit, le fait que cette propriété soit utilisée un demi-point de pourcentage de plus sur les ordinateurs de bureau que sur les téléphones mobiles est inattendu, et 7 % semble beaucoup plus élevé que ce que j'aurais prédit.
+
+## Les polices variables sont l'avenir
+
+[Les polices variables](https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide) permettent d'inclure plusieurs poids et styles de police dans un seul fichier.
+
+{{ figure_markup(
+  caption="Pourcentage des pages qui incluent une police variable.",
+  content="1.8%",
```

Comment:
Remarque générale => harmoniser, les chiffres utilisent tantôt le point ou la virgule, let's go pour la virgule :)

Reply:
I'm not sure what you mean here. Could you suggest the change needed?

Reply:
Sometimes we're using the dot (1.8%), sometimes the coma (1,8%) as separator. I guess we should harmonize, in french, it's supposed to be the coma.

Reply:
Ah gotta. Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2114/commits/58e1718e7df6fbeeeb04338cf14501f032fffda6